include/assimp/mesh.h

```diff
@@ -1007,41 +1025,53 @@ struct aiSkeletonBone {
         // empty
     }
 
+    /// @brief The class destructor.
     ~aiSkeletonBone() {
         delete[] mWeights;
         mWeights = nullptr;
     }
 #endif // __cplusplus
 };
 /**
- *  @brief
+ * @brief A skeleton represents the bone hierarchy of an animation.
+ *
+ * Skeleton animations can be described as a tree of bones:
+ *                  root
```

Comment:
It might be good to be explicit which mBones bone is the root in this doc. Since the aiSkeletonBone constructor sets the mParent for all new aiSkeletonBone's to -1, it is hard to know how to determine the root node. Is it the first node in the mBones?

Reply:
Good question. Initially the root node will not be set when you have to use the default constructor. I will introduce another constructor where you can explicitly specify which root node shall be used.